integrations/pytorch-lightning/Neptune-PyTorch-Lightning-basic.ipynb

```diff
@@ -329,6 +329,26 @@
     "trainer.fit(model, train_loader)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Stop Neptune logger at the end"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
@aniezurawski this is the only notebook where things actually get changed -> and different cell types are added to cells. 
In this case it's code but it can also be tests, header, comment and others. 

Once this is changed from the main directory you run python ci/build.py to create the rest of them.